content/docs/authorization/troubleshooting.md

```diff
@@ -16,6 +16,27 @@ Description: >
 
 ---
 
+### The Failure of Building an Authorization RPM
```

Comment:
We link to the topics associated with RPM vs Helm Deployment here https://dell.github.io/csm-docs/docs/authorization/troubleshooting/#rpm-deployment - can you add your title in the RPM section so it's easy to see and access

Reply:
done